### ISSUE TITLE
Unable to replicate results on BOOLEAN SATISFIABILITY PROBLEM

### ISSUE THREAD

**Issue by mukurgupta** [HKUNLP/diffusion-vs-ar#8]
Thanks for sharing the code. 

I am not able to replicate the results on BOOLEAN SATISFIABILITY PROBLEM reported in Figure 4 of the paper. I use the provided bash script ```scripts/3-sat/train-mdm.sh``` with the 3sat5 dataset. After tuning the learning rate, the max accuracy I get is ~47%. However much higher number is reported in the paper. Can you please help me with this? 

Thank you.
